torch/csrc/distributed/rpc/rpc_agent.cpp

```diff
@@ -4,6 +4,8 @@ namespace torch {
 namespace distributed {
 namespace rpc {
 
+const size_t WorkerId::MAX_NAME_LEN=128;
```

Comment:
nit: can use contexpr inline in the header file.

Reply:
cannot use inline var though, that's C++17